core/kernel/classes/class.Resource.php

```diff
@@ -578,9 +578,16 @@ public function duplicate($excludedProperties = [])
      */
     public function delete($deleteReference = false)
     {
+        try {
+            $parentClass = $this->getOnePropertyValue($this->getProperty(OntologyRdf::RDF_TYPE));
```

Comment:
Is this the class that the user select to delete or the parent class? In the ticket we need the one that user selected to delete too.